src/interfaces/correction-filing-interface.ts

```diff
@@ -24,8 +24,5 @@ export interface CorrectionFilingIF {
     correctedFilingDate: string
     type?: CorrectionTypes
     comment: string
-  },
-  incorporationApplication?: {},
-  changeOfRegistration?: {},
-  registration?: {}
```

Comment:
@lekshmimallika-aot , compare with [correction schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/correction.json).

Reply:
@lekshmimallika-aot I saw these fields are being used in the backend, but I didn't dig deeper. What bothers me are the fields related to these objects that are used else where.

Reply:
Is this old BE code that will be updated by https://app.zenhub.com/workspace/o/bcgov/entity/issues/13330 ?

Reply:
@lewischenstudio I already removed the corrected filing object from a correction filing in [13192](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/13192). This PR is to clean up Filings UI as well.